paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -322,13 +322,13 @@ internal abstract class BaseSheetViewModel(
 
     private fun reportPaymentSheetShown(currentScreen: PaymentSheetScreen) {
         when (currentScreen) {
-            is PaymentSheetScreen.Loading, AddAnotherPaymentMethod, is PaymentSheetScreen.EditPaymentMethod -> {
+            is PaymentSheetScreen.Loading, is PaymentSheetScreen.EditPaymentMethod -> {
```

Comment:
My understanding was that we only log the _first_ screen that PaymentSheet displays. That’s why `AddAnotherPaymentMethod` was never logged, as it can’t be the first screen by definition. Has this changed?

Reply:
Correct! But `FlowController` can show `AddAnotherPaymentMethod` as the first screen if there is inputted data that hasn't been saved. `iOS` at least has been reporting it this way.

Reply:
Gotcha, thanks for aligning them then!